homeassistant/components/mqtt/tag.py

```diff
@@ -102,7 +113,7 @@ def __init__(
         self.discovery_data = discovery_data
         self.hass = hass
         self._sub_state: dict[str, EntitySubscription] | None = None
-        self._value_template = MqttValueTemplate(
+        self._value_template: Callable[..., ReceivePayloadType] = MqttValueTemplate(
```

Comment:
```suggestion
        self._value_template: Callable[[ReceivePayloadType], ReceivePayloadType]
        self._value_template = MqttValueTemplate(
```

Reply:
The value template has a second string var (default)